### PR TITLE
Tidy banlist & rotation indicators on the card page

### PR DESCRIPTION
**Pull request by plural**
Building on noah's awesome updates to the card page, add rotation (standard cardpool) and current standard banlist legality to the right column box.

Addresses #214  and #448 .

 
Hide banlist history unless the user asks for it.

Zer0 current:
Screenshot 2021-03-12 at 9 22 23 AM

New Zer0 (default & expanded)
Screenshot 2021-03-12 at 9 20 14 AM
Screenshot 2021-03-12 at 9 20 28 AM

Cards with multiple printings did a funny thing with MWL entries, so i had to change that a bit and update the factinon id page and the deckbuilder pick your id pages to roll with this.

### CODE REVIEW

File: app/Resources/views/Search/display-zoom.html.twig [hunk 1]
```diff
@@ -117,6 +117,67 @@
                 <a href="{{ path('cards_zoom',{_locale:app.request.locale, 'card_code':card.code}) }}"> English </a>
             </div>
 
+            <div class="panel-body">
+                <table>
+                    <thead>
+                        <tr>
+                            <th>
+{% set is_ineligible = false %}
```
Comment (NoahTheDuke) [inline app/Resources/views/Search/display-zoom.html.twig]: indentation here is wacky lol

File: app/Resources/views/Search/display-zoom.html.twig [hunk 2]
```diff
@@ -117,6 +117,67 @@
...
+                       {% for mwl_info in card.mwl_info %}
+                         {% if mwl_info.active %}
+                              {% if mwl_info.no_effect %}<span title="No restrictions">✅</span>{% endif %}
+                              {% if mwl_info.is_restricted == true %}<span title="Restricted">🦄{% endif %}
+                              {% if mwl_info.deck_limit is not null and mwl_info.deck_limit == 0 %}<span title="Banned">🚫</span>{% endif %}
+                              {% if mwl_info.universal_faction_cost == 1 %}<span title="1 additional Universal Influence">1️⃣</span>{% endif %}
+                              {% if mwl_info.universal_faction_cost == 3 %}<span title="3 additional Universal Influence">3️⃣</span>{% endif %}
+                          {% endif %}
+                        {% endfor %} : Standard Ban List
+						<a id="mwl-history" href="" onClick="
```
Comment (NoahTheDuke) [inline app/Resources/views/Search/display-zoom.html.twig]: can this function be moved to javascript somewhere and just called here?

File: src/AppBundle/Controller/SearchController.php [hunk 3]
```diff
@@ -538,8 +539,15 @@ public function displayAction(
             $title = $q;
         }
 
+		$currentRotationCycles = [];
```
Comment (NoahTheDuke) [inline src/AppBundle/Controller/SearchController.php]: looks like you're mixing spaces and tabs here

File: src/AppBundle/Controller/SearchController.php [hunk 4]
```diff
@@ -538,8 +539,15 @@ public function displayAction(
...
+
         if ($view == "zoom") {
             $card = $cards[0];
+			$rotationService = new RotationService($entityManager);
```
Comment (NoahTheDuke) [inline src/AppBundle/Controller/SearchController.php]: tabs here too

File: src/AppBundle/Service/CardsData.php [hunk 5]
```diff
@@ -879,13 +879,24 @@ public function get_mwl_info(array $cards)
                     $deck_limit = $card_mwl['deck_limit'] ?? null;
                     // Ceux-ci signifient la même chose
                     $universal_faction_cost = $card_mwl['universal_faction_cost'] ?? $card_mwl['global_penalty'] ?? 0;
-                    $response[] = [
+                    $response[$mwl->getName()] = [
                         'mwl_name'               => $mwl->getName(),
                         'active'                 => $mwl->getActive(),
+                        'no_effect'              => false,
```
Comment (NoahTheDuke) [inline src/AppBundle/Service/CardsData.php]: If you use `elseif` above in the twig files, you could have the `else` branch do this for you.

File: src/AppBundle/Controller/SearchController.php (hunk 3 above)
Comment (plural) [reply src/AppBundle/Controller/SearchController.php]: done

File: src/AppBundle/Controller/SearchController.php (hunk 4 above)
Comment (plural) [reply src/AppBundle/Controller/SearchController.php]: done

File: app/Resources/views/Search/display-zoom.html.twig (hunk 1 above)
Comment (plural) [reply app/Resources/views/Search/display-zoom.html.twig]: done

File: app/Resources/views/Search/display-zoom.html.twig (hunk 2 above)
Comment (plural) [reply app/Resources/views/Search/display-zoom.html.twig]: moved to CSS instead of JS.

File: src/AppBundle/Service/CardsData.php (hunk 5 above)
Comment (plural) [reply src/AppBundle/Service/CardsData.php]: yep, no longer needed.

**Comment by plural** [comment]
ok, much tidying done.

**Review by NoahTheDuke — APPROVED** [review]
Looks great

**Pull request merged**